code/src/main/resources/site/i18n/phrases_no.properties

```diff
@@ -6,35 +6,50 @@ site.testmode-help = Aktiver testmodus hvis du alltid vil vise forespørselsboks
 site.identifier = Identifikator
 site.identifier-help = Skriv inn tekst og/eller nummer for å identifisere gjeldende forespørsel. Alle skjemaer som sendes inn av besøkende vil bli koblet til denne identifikatoren. Når du vil opprette en ny forespørsel, endre identifikatoren til noe nytt og unikt. Vennligst ikke bruk mellomrom eller spesialtegn annet enn "-".
 site.theme = Fargetema
+site.theme-help = Velg et fargetema for forespørselsskjemaet.
 site.theme.light = Lyst tema
 site.theme.light-minimal = Lyst tema som arver nettsidens styling
 site.theme.dark = Mørkt tema
 site.theme.dark-minimal = Mørkt tema som arver nettsidens styling
 site.fontSize = Grunn font-size (px)
 site.fontSize-help = Som standard vil font-size arves fra body. Bruk kun dette feltet til å angi en font-size manuelt om nødvendig (skriv inn verdi mellom 1 og 30).
-site.box = Boks
+site.box = Forespørselboks
 site.boxTitle = Tittel
+site.boxTitle-help = Legg til en tittel på forespørselsboksen, som lar bruker gå videre til spørreskjemaet. For eksempel: Fortell oss hva du synes!
 site.boxText = Tekst
-site.boxContinue = Knapp
+site.boxText-help = Legg til en beskrivende tekst på forespørselsboksen. Den vil bli vist under tittelen.
+site.boxContinue = Knappetekst
+site.boxContinue-help = Dette er knappen en bruker kommer til å klikke på dersom de �nsker å gå videre til spørreskjemaet. Legg til en passende knappetekst.
```

Comment:
Seeing � here?

Reply:
When you build and install the app or in the code? I got � in the app config if i did not replace ø and å with Ã¥ and Ã¸ 🤔

Reply:
Is it possible your IDE is saving files in ANSI, ASCII or some other encoding instead of UTF-8? 🤔

Reply:
Yeah, that was it :P Better now?

Reply:
Perfect 👌